package.json

```diff
@@ -35,15 +35,14 @@
     "rollup-pluginutils": "^1.5.0"
   },
   "peerDependencies": {
-    "babel-core": "6 || 7 || ^7.0.0-alpha || ^7.0.0-beta || ^7.0.0-rc"
+    "@babel/core": "7 || ^7.0.0-beta"
```

Comment:
i dont really like allowing stable 7 here, as we do not have yet knowledge if it will be compatible with this version of the rollup-plugin-babel's code, but I guess we are doing it everywhere else, so ¯\_(ツ)_/¯